geetools/ee_dictionary.py

```diff
@@ -80,3 +82,116 @@ def getMany(self, list: list | ee.List) -> ee.List:
+           keys will be saved in the column "system:index" and the values in the
+           column "value".
+
+        2. dict: when values are a ee.Dictionary, the keys will be saved in the
+           column "system:index" and the values will be treated as each
+           Feature's properties.
+
+        3. list: when values are a ee.List of numbers or strings, the keys will
+           be saved in the column "system:index" and the values in as many
+           columns as items in the list. The column name pattern is "value_{i}"
+           where i is the position of the element in the list.
+
+        These are the only supported patterns. Other patterns should be converted
+        to one of these. For example, the values of a reduction using the
+        reducer `ee.Reducer.frequencyHistogram()` are of type `ee.Array` and
+        the array contains lists.
+
+        Examples:
+            .. code-block:: python
+
+                import ee, geetools
+
+                ee.Initialize()
+
+                d = ee.Dictionary({"foo": 1, "bar": 2})
+                d.geetools.toTable().getInfo()
+
+            .. code-block:: python
+
+                import ee, geetools
+
+                ee.Initialize()
+
+                d = ee.Dictionary({
+                  "Argentina": {"ADM0_CODE": 12, "Shape_Area": 278.289196625},
+                  "Armenia": {"ADM0_CODE": 13, "Shape_Area": 3.13783139285},
+                })
+                d.geetools.toTable('dict').getInfo()
+
+            .. code-block:: python
+
+                import ee, geetools
+
+                ee.Initialize()
+
+                d = ee.Dictionary({
+                  "Argentina": [12, 278.289196625],
+                  "Armenia": [13, 3.13783139285],
+                })
+                d.geetools.toTable().getInfo()
+
+            .. code-block:: python
+
+                import ee, geetools
+
+                ee.Initialize()
+
+                # reduction
+                ran = ee.Image.random().multiply(10).reduceRegion(
+                    reducer=ee.Reducer.fixedHistogram(0, 1.1, 11),
+                    geometry=ee.Geometry.Point([0,0]).buffer(1000),
+                    scale=100
+                )
+                # process to get desired format
+                res = ee.Array(ee.Dictionary(ran).get('random'))
+                reslist = res.toList()
+                keys = reslist.map(lambda i: ee.Number(ee.List(i).get(0)).multiply(100).toInt().format())
+                values = reslist.map(lambda i: ee.Number(ee.List(i).get(1)).toInt())
+                final = ee.Dictionary.fromLists(keys, values)
+                # fetch
+                final.geetools.toTable().getInfo()
+
+        Parameters:
+            valueType: this will define how to process the values.
+
+        Returns:
+            a ee.FeatureCollection in which the keys of the ee.Dictionary are
+            in the `system:index` and the values are in new columns.
```

Comment:
type is already written in the proptotye no need to remind it here.

```suggestion
            a collection in which the keys of the :py:class:`ee.Dictionary` are
            in the ``system:index`` and the values are in new columns.
```